apps/editing-toolkit/webpack.config.js

```diff
@@ -29,17 +28,17 @@ const isDevelopment = process.env.NODE_ENV !== 'production';
  * @see {@link https://webpack.js.org/api/cli/}
  *
  * @param   {object}  env                           environment options
+ * @param   {string}  env.source                    plugin slugs, comma separated list
  * @param   {object}  argv                          options map
- * @param   {string}  argv.source                   plugin slug
  * @param   {string}  argv.entry                    entry path
  * @returns {object}                                webpack config
  */
-function getWebpackConfig( env = {}, argv = {} ) {
+function getWebpackConfig( env = { source: '' }, argv = {} ) {
```

Comment:
The defaulting of `source` has some loopholes: if I call `getWebpackConfig( { nosource: 'gotcha' } )`, the `env.source.split` few lines below will access an undefined field.